internal/server/tile_handler.go

```diff
@@ -70,9 +73,21 @@ func newTileHandler(handler defaultHandler) (tileHandler, error) {
 	}, errors.Join(err1, err2, err3, err4)
 }
 
+func (h *tileHandler) reloadEntities(newEntities reloadableEntities) {
+	slog.WarnContext(pkg.BackgroundContext(), "Requesting to refresh entities from configuration")
+	h.mux.Lock()
```

Comment:
Do I really need this mutex?  I put it in thinking I'd need to do multiple setters but given it ended up being just one, that should be atomic right?